pkg/reconciler/v1alpha1/autoscaling/kpa/kpa_test.go

```diff
@@ -42,29 +42,39 @@ import (
 )
 
 var (
-	gracePeriod = 60 * time.Second
-	idlePeriod  = 9 * time.Minute
+	gracePeriod   = 60 * time.Second
+	idlePeriod    = 9 * time.Minute
+	configMapData = map[string]string{
+		"max-scale-up-rate":                       "1.0",
+		"container-concurrency-target-percentage": "0.5",
+		"container-concurrency-target-default":    "10.0",
+		"stable-window":                           "5m",
+		"panic-window":                            "10s",
+		"scale-to-zero-grace-period":              gracePeriod.String(),
+		"tick-interval":                           "2s",
+	}
 )
 
 func newConfigWatcher() configmap.Watcher {
-	return configmap.NewStaticWatcher(
-		&corev1.ConfigMap{
-			ObjectMeta: metav1.ObjectMeta{
-				Namespace: system.Namespace,
-				Name:      autoscaler.ConfigName,
-			},
-			Data: map[string]string{
-				"max-scale-up-rate":                       "1.0",
-				"container-concurrency-target-percentage": "0.5",
-				"container-concurrency-target-default":    "10.0",
-				"stable-window":                           "5m",
-				"panic-window":                            "10s",
-				"scale-to-zero-grace-period":              gracePeriod.String(),
-				"tick-interval":                           "2s",
-			},
-		})
+	return configmap.NewStaticWatcher(&corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: system.Namespace,
+			Name:      autoscaler.ConfigName,
+		},
+		Data: configMapData,
+	})
+}
+
+func newDynamicConfig(t *testing.T) *autoscaler.DynamicConfig {
+	dynConfig, err := autoscaler.NewDynamicConfigFromMap(configMapData, TestLogger(t))
+	if err != nil {
+		t.Errorf("Error creating DynamicConfig: %v", err)
+	}
+	return dynConfig
 }
 
+// TODO(josephburnett): Convert KPA tests to table tests.
```

Comment:
This is hard because of the fake resource :-/

Reply:
Yeah.  :(  I still want to leave this TODO in order to figure it out.  This is *way* too verbose.